ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 - agent: kubernetes
-  cluster: build03
+  cluster: build01
```

Comment:
Are those changes expected ?

Reply:
Just did rebase and make jobs, no idea why it happened.
Redo and it looks good now.